app/mailers/ctc_signup_mailer.rb

```diff
@@ -10,4 +10,14 @@ def beta_navigation(email_address:, name:)
       subject: @subject,
     )
   end
+
+  def launch_announcement(email_address:, name:)
+    @name = name
+    subject = "Thank you for signing up to receive updates regarding GetCTC! / ¡Gracias por registrarse para recibir las actualizaciones de GetCTC!"
```

Comment:
do we not want people to open this email 😆

Reply:
Good question. :) Kelly said it was fine!! :) I'll call and double-check.

Reply:
Shorter would be better but in this case for expediency Kelly says we can leave it the way it is.